clearwater-snmpd/usr/share/clearwater/infrastructure/scripts/snmpd_fd_handler

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright (C) 2018  Metaswitch Networks Ltd
+# Handles SNMP requests offloaded by snmpd for the file descriptor OIDs
+oid_prefix=.1.2.826.0.1.1578918.17.1
+oid_req=$2
+
+if [ "$1" = "-n" ]; then
```

Comment:
What is actually calling this script? Feels like we should have a comment here explaining how we expect it to be called, and what arguments are permissible. Also, are you sure this string comparison works? I know these are a pain in bash, and thought they needed `==`, and possibly double square brackets. 
Happy if it works though